src/pages/index.tsx

```diff
@@ -1,63 +1,23 @@
 import React, { useEffect, useState } from "react";
-import { GetStaticProps } from "next";
 import Router from "next/router";
 import Link from 'next/link';
 
 import { Bubble, Label, Comment } from "@prisma/client";
-import prisma from '../../prisma/client';
+import useFetch from "../hooks/useFetch";
 
 import Header from '../components/Header/Header';
 import BubbleListItem from "../components/BubbleListItem/BubbleListItem";
 import FloatingButton from '../components/FloatingButton/FloatingButton';
-import BubbleDetails from "../components/BubbleDetailsModal/BubbleDetailsModal";
 import NewBubbleModal from "../components/NewBubbleModal/NewBubbleModal";
+import BubbleDetailsModal from "../components/BubbleDetailsModal/BubbleDetailsModal";
 
 import postBubbles from '../services/postBubbles';
 import postComments from '../services/postComments';
 import postLabels from '../services/postLabels';
 import alteredLabels from '../services/alteredLabels';
 
 import styles from './_home.module.css';
-
-export const getStaticProps: GetStaticProps = async () => {
-  const bubblesResponse = await prisma.bubble.findMany({
-    include: {
-      labels: true,
-      comments: {
-        include: {
-          author: {
-            select: {
-              avatarUrl: true,
-              name: true,
-            },
-          },
-        },
-      },
-      author: {
-        select: {
-          avatarUrl: true,
-        },
-      },
-    },
-  });
-
-  const labels = await prisma.label.findMany();
-
-  const serializableBubbles = bubblesResponse.map(bubble => ({
-    ...bubble,
-    createdAt: bubble.createdAt.toString(),
-
-    comments: bubble.comments.map(comment => ({
-      ...comment,
-      createdAt: comment.createdAt.toString(),
-    })),
-  }));
-  
-  return {
-    props: { bubbles: serializableBubbles, labels },
-    revalidate: 1,
-  };
-};
```

Comment:
Então man, se vc faz isso vc está removendo totalmente a vantagem do next no first paint. Vi que vc colocou esse código no /api/bubbles/index. Agora tem que bater lá e trazer isso no getStaticProps. Tem que ser o primeiro conteúdo antes do SWR bater na api

Reply:
Sim... estou tentando fazer com o api.get mas da erro no axios... e não to entendendo esse erro...